ulog/log.go

```diff
@@ -51,6 +53,9 @@ type Log interface {
 	// RawLogger returns underlying logger implementation (zap.Logger) to get around the ulog.Log interface
 	RawLogger() zap.Logger
 
+	// Sentry returns the underlying sentry hook
+	Sentry() *sentry.Hook
```

Comment:
Why don't we do unit (e.g. package-level) tests instead then? Adding something to our interface purely for a test seems like a bad idea, since we'll have to support it. It's very weird to me that the logger interface has a public member that's tied to a **sentry** thing.